README.md

```diff
@@ -39,5 +39,10 @@ Under File -> Settings -> Editor -> Code Style import the google-styleguide (gea
 2. Create gradle.properties file (it will overwrite project's properties)
 3. Add `ivrApiKey=<voto-api-key>` to set VOTO communication.
 
+## Load locations locally when starting the server
+1. Set `loadLocations=true` in project or home `gradle.properties` file.
```

Comment:
this is not necessary, you can run: ./gradlew clean bootrun -PloadLocations=true

Reply:
Fantastic, I tried without clean. Changing it in readme